docs/guides/buckets.mdx

```diff
@@ -0,0 +1,338 @@
+---
+sidebar_position: 3
+title: Buckets
+description: Complete guide to Buckets in ReductStore
+---
+
+<head>
+  <link rel="canonical" href="https://www.reduct.store/docs/guides/buckets"/>
+</head>
+
+import CodeBlock from "@theme/CodeBlock";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Buckets
+
+## Concepts
+
+A bucket is a container used for data storage and serves as a logical grouping of data. It stores time-series data, with each entry having a unique name. If you're familiar with databases, think of a bucket as a database, and entries as tables.
```

Comment:
We can explain slightly more the concept instead of "If you're familiar with databases, think of a bucket as a database, and entries as tables.": 

Each bucket can contain multiple entries, with each entry representing a time-series dataset composed of time-stamped records. In addition, each bucket can be fully secured with unique authentication tokens (passwords).